### ISSUE TITLE
Change OSX setup-cfgs to be minor version agnostic

### ISSUE THREAD

**Issue by matt-chan** [theochem/horton#98]
OSX updates too frequently. Instead of checking against Darwin-10.10.0 etc, we should just check against 10.10, 10.11, etc.

**Comment by sfias** [comment]
how about this in setup.py:
```
def detect_machine():
    '''Return a description of the machine name, used for data/setup_cfgs/...'''
    if sys.platform == 'linux2':
        dist = platform.linux_distribution()
        return ('Linux-%s-%s-%s' % (dist[0], dist[1], platform.machine())).replace(' ', '_')
    elif sys.platform == 'darwin':
        mac_ver = platform.mac_ver()
        mac_os  = mac_ver[0].rpartition('.')
        return 'Darwin-%s-%s' % (mac_os[0], mac_ver[2])
    else:
        return 'unknown'
```
and just use setup.Darwin-10.11-x86_64.cfg as file... This works on my mac.

**Comment by tovrstra** [comment]
Fixed by merging #104.
